man/systemd-stub.xml

```diff
@@ -185,7 +185,7 @@
       configuration.</para>
 
       <para>In case Secure Boot is enabled, these files will be validated using keys in UEFI DB, Shim's DB or
-      Shim's MOK, and will be rejected otherwise. Additionally, if the both the addon and the UKI contain a a
+      Shim's MOK, and will be rejected otherwise. Additionally, if both the addon and the UKI contain a a
```

Comment:
If you touch this, one more typo to fix:

```suggestion
      Shim's MOK, and will be rejected otherwise. Additionally, if both the addon and the UKI contain a
```

Reply:
Thanks, I don't know how I could have missed this.... my eyes are failing :D